{
    "checksum": "c6d4d9670e950bbc36183b6ffc6c3f91d94e06c2e4252196df28849ba1f092c5",
    "file": "wrong-file-sha384-hash-in-tar-gz-archive.conf",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 6,
    "package": "backupchecker",
    "path": "backupchecker/1.7-3/functional-tests/wrong-file-sha384-hash-in-tar-gz-archive/wrong-file-sha384-hash-in-tar-gz-archive.conf",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/backupchecker/1.7-3/",
        "metric": {
            "size": 6452
        },
        "pts_link": "https://tracker.debian.org/pkg/backupchecker",
        "sloc": [
            [
                "python",
                8568
            ],
            [
                "makefile",
                161
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/backupchecker",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/backupchecker/1.7-3/functional-tests/wrong-file-sha384-hash-in-tar-gz-archive/wrong-file-sha384-hash-in-tar-gz-archive.conf",
    "stat": {
        "perms": "rw-r--r--",
        "size": 177,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.7-3"
}